src/iop/crop.c

```diff
@@ -1111,6 +1116,35 @@ static gchar *_aspect_format(gchar *original,
     return g_strdup_printf("%s  %4.2f", original, (float)adim / (float)bdim);
 }
 
+static void _crop_handle_flip(dt_iop_module_t *self, const dt_image_orientation_t mode)
+{
+  dt_iop_crop_params_t *p = self ? self->params : NULL;
+  if(!p) return;
```

Comment:
`if(!p || (!p->cx && !p->cy && p->cw == 1.f && p->ch == 1.f)) return;`

Reply:
This tricky one seems good to me :-)

Reply:
We can check for fully symmetrical here as an easy alternative